npm-shrinkwrap.json

```diff
@@ -446,51 +460,54 @@
     },
     "nodemailer-smtp-transport": {
       "version": "0.1.13",
-      "from": "nodemailer-smtp-transport@^0.1.12",
+      "from": "https://registry.npmjs.org/nodemailer-smtp-transport/-/nodemailer-smtp-transport-0.1.13.tgz",
       "resolved": "https://registry.npmjs.org/nodemailer-smtp-transport/-/nodemailer-smtp-transport-0.1.13.tgz",
       "dependencies": {
         "nodemailer-wellknown": {
           "version": "0.1.4",
-          "from": "nodemailer-wellknown@^0.1.1",
+          "from": "https://registry.npmjs.org/nodemailer-wellknown/-/nodemailer-wellknown-0.1.4.tgz",
           "resolved": "https://registry.npmjs.org/nodemailer-wellknown/-/nodemailer-wellknown-0.1.4.tgz"
         },
         "smtp-connection": {
           "version": "1.1.0",
-          "from": "smtp-connection@^1.0.0",
+          "from": "https://registry.npmjs.org/smtp-connection/-/smtp-connection-1.1.0.tgz",
           "resolved": "https://registry.npmjs.org/smtp-connection/-/smtp-connection-1.1.0.tgz"
         }
       }
     },
     "request": {
       "version": "2.51.0",
-      "from": "request@^2.39.0",
+      "from": "https://registry.npmjs.org/request/-/request-2.51.0.tgz",
       "resolved": "https://registry.npmjs.org/request/-/request-2.51.0.tgz",
       "dependencies": {
         "bl": {
           "version": "0.9.3",
-          "from": "bl@~0.9.0",
+          "from": "bl@0.9.3",
+          "resolved": "https://registry.npmjs.org/bl/-/bl-0.9.3.tgz",
           "dependencies": {
             "readable-stream": {
               "version": "1.0.33",
-              "from": "readable-stream@~1.0.26",
+              "from": "readable-stream@1.0.33",
+              "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.33.tgz",
               "dependencies": {
                 "core-util-is": {
                   "version": "1.0.1",
-                  "from": "core-util-is@~1.0.0",
+                  "from": "https://registry.npmjs.org/core-util-is/-/core-util-is-1.0.1.tgz",
                   "resolved": "https://registry.npmjs.org/core-util-is/-/core-util-is-1.0.1.tgz"
                 },
                 "isarray": {
                   "version": "0.0.1",
-                  "from": "isarray@0.0.1",
+                  "from": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz",
                   "resolved": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz"
                 },
                 "string_decoder": {
                   "version": "0.10.31",
-                  "from": "string_decoder@~0.10.x"
+                  "from": "string_decoder@0.10.31",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.